examples/get-insights-data/main.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+import csv
+from utils.utils import PublicAPIClient, get_commandline_arguments
+
+
+def main():
+    args = get_commandline_arguments(
+        [
+            (
+                ["course_id"], {"type": int, "help": "id of the course to get insights for"},
+            )
+        ]
+    )
+    public_api_client = PublicAPIClient(args.subdomain)
+    perspectives = fetch_course_perspectives(public_api_client, args.course_id)
+    course_data = fetch_course_data(public_api_client, args.course_id)
+
+    construct_summary(public_api_client, perspectives, course_data)
+    construct_student_insights(public_api_client, perspectives, course_data)
+
+
+def fetch_course_perspectives(public_api_client, course_id):
+    response = public_api_client.request("get", f"/courses/{course_id}/perspectives")
```

Comment:
I doesn't matter much, but please remove the leading slash from all `public_api_client.request` calls. We have `/api/public/v1/{url}` in utils.py, so it a leading slash will lead to a double slash.

Reply:
done